### PR TITLE
fcct: point to the container image

### PR DESCRIPTION
**Pull request by lucab**
This tweaks the FCCT docpage, prominently pointing to the container image.

### CODE REVIEW

File: modules/ROOT/pages/producing-ign.adoc [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 == A simple example
+
 Create a basic Ignition file that modifies the default FCOS user `core` to allow this user login with an SSH key.
 
 The overall steps are as follows:
 
 . Write the Fedora CoreOS Configuration (FCC) file in the YAML format.
-. Download and install the Fedora CoreOS Configuration Transpiler (`fcct`).
-. Use `fcct` to convert the FCC file into an Ignition (JSON) file.
+. Use the `quay.io/coreos/fcct:release` container image to convert the FCC file into an Ignition (JSON) file.
```
Comment (jlebon) [inline modules/ROOT/pages/producing-ign.adoc:18]: Maybe in parentheses here, we could add " (on Fedora, you can also install the `fcct` package)" or something?

Comment (lucab) [reply modules/ROOT/pages/producing-ign.adoc:18]: Hmm, after your comment I realized we already have a dedicated page for getting `fcct` https://docs.fedoraproject.org/en-US/fedora-coreos/using-fcct/.

I think there is some room to deduplicate the content and merge the two pages.

Comment (jlebon) [reply modules/ROOT/pages/producing-ign.adoc:18]: Yeah, probably worth just linking to the docs page and adding the `fcct` package there as another way for users who already have a Fedora environment?

**Pull request merged**